examples/search_examples/public/search/app.tsx

```diff
@@ -721,6 +765,10 @@ export const SearchExamplesApp = ({
                 strategy. This request does not take the configuration of{' '}
                 <EuiCode>TopNavMenu</EuiCode> into account, but you could pass those down to the
                 server as well.
+                <br />
+                When executing search on the server, make sure to cancel the search in case user
```

Comment:
Is it worth mentioning that we have the same `AbortController` API on the client and server?

Reply:
Adding 👍